### PR TITLE
Fruitdetail

### PR DESCRIPTION
**Pull request by carodriguezp**

### CODE REVIEW

**Comment by davepadrino** [comment]
is this ready for review?

File: src/App.style.ts [hunk 1]
```diff
@@ -112,4 +113,73 @@ export const StyledFormGroup = styled(FormGroup)`
...
 `
+export const StyledDetail = styled.div`
+    position: relative;
+    top: 40px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 20px;
+    padding: 10px;
+    background: rgba(219, 212, 60, 0.76);
```
Comment (davepadrino) [inline src/App.style.ts]: above you're using hex for colors, and here rgb, please unify styles to be consistent and lets use one of them.

File: src/App.style.ts [hunk 2]
```diff
@@ -112,4 +113,73 @@ export const StyledFormGroup = styled(FormGroup)`
...
+  }
+
+  td{
+    font-size: 1.6rem;
+  }
+
+  @media all and (min-width: 768px) {
+    margin: 15px;
+  }
+`
```
Comment (davepadrino) [inline src/App.style.ts]: missing whitespace

**Review by davepadrino — APPROVED** [review]

File: src/App.style.ts (hunk 1 above)
Comment (carodriguezp) [reply src/App.style.ts]: unified

**Review by davepadrinoxing — APPROVED** [review]

**Pull request merged**